test/eigen.jl

```diff
@@ -102,7 +104,7 @@
         v = randn(SVector{3,Float64})
         m = v*v'
         vv = sum(abs2, v)
-        vals, vecs = eig(m)::Tuple{SVector,SMatrix}
+        vals, vecs = eigen(m)
```

Comment:
`::Tuple{SVector,SMatrix}` was removed. How can we be sure this is not calling generic code?